examples/trivial.go

```diff
@@ -45,18 +44,12 @@ func (r *LoggingRecorder) RecordSpan(span lightstep.RawSpan) {
 }
 
 func main() {
-	flag.Parse()
-	if len(*accessToken) == 0 {
-		fmt.Println("You must specify --access_token")
-		os.Exit(1)
-	}
-
 	loggableRecorder := &LoggingRecorder{}
 
 	// Use LightStep as the global OpenTracing Tracer.
 	opentracing.InitGlobalTracer(lightstep.NewTracer(lightstep.Options{
 		AccessToken: *accessToken,
-		Collector:   lightstep.Endpoint{Host: "localhost", Port: 9997, Plaintext: true},
+		Collector:   lightstep.Endpoint{Host: "localhost", Port: 8360, Plaintext: true},
```

Comment:
Can we separate this into a separate PR?

Reply:
See https://github.com/lightstep/lightstep-tracer-go/pull/191